api/internal/owner/views.py

```diff
@@ -113,8 +113,18 @@ def update_billing_address(self, request, *args, **kwargs):
         if not billing_address:
             raise ValidationError(detail="No billing_address sent")
         owner = self.get_object()
+
+        formatted_address = {
+            "line1": billing_address["line_1"],
```

Comment:
Why are line1/2 camel? Shouldn't it be snake?

Reply:
ah yeah this is what I was mentioning on the applications channel; stripe considers line1 and line2 to be the correct key format instead of line_1 and line_2, which is different than our snakeify method. This seemed to be the most painless way to isolate that "rewrapping"